app/controllers/organizations_controller.rb

```diff
@@ -71,10 +71,40 @@ def feedback
     @wishlists = @organization.wishlists
     authorize @organization
     @wish = Wish.new
-    @wishes = @organization.wishes.order(created_at: :desc)
-    return unless params[:filter].present?
-
-    @wishes = Wishes::FeedbackFilterer.new(filter_params:, scope: @wishes).call
+    @wishes = @organization.wishes
```

Comment:
Do we need all these comments?

Reply:
Also why do we need this line if we assign @wishes right after without a conditional?